partcad-cli/src/partcad_cli/click/commands/add/part.py

```
@@ -46,12 +47,18 @@
-        if desc:
-            config["desc"] = desc
-        if provider:
-            config["provider"] = provider
-        if prj.add_part(kind, path, config):
-            Path(path).touch()
+    """
+    CLI command to add a part to the project without copying.
+    """
+    project = ctx.get_project(pc.ROOT)
+    if not project:
+        raise click.UsageError("Failed to retrieve the project.")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)